### ISSUE TITLE
Can you expand the lrp.py to cover BigBird?

### ISSUE THREAD

**Issue by ngoel17** [hila-chefer/Transformer-Explainability#43]
Most of the NLP tasks use BigBird due to long text support. Can you please extend the class to support BigBird, by adding rules for torch.nn.modules.sparse.Embedding etc?

**Comment by hila-chefer** [comment]
Hi @ngoel17, thank you for your interest!
We currently do not have plans to extend our implementation to BigBird, however, I'd suggest using [the method from our second paper](https://github.com/hila-chefer/Transformer-MM-Explainability) which does not require LRP (only gradient propagation).

Best,
Hila.
